app/subscriber/src/store/hooks/subscriber/useMinisters.ts

```diff
@@ -9,23 +9,24 @@ interface IMinisterController {
   getMinisters: () => Promise<IMinisterModel[]>;
 }
 
-export const useMinisters = (): [IProfileState, IMinisterController] => {
+export const useMinisters = (): [IMinisterModel[], IMinisterController] => {
   const api = useApiSubscriberMinisters();
   const dispatch = useAjaxWrapper();
-  const [state, store] = useProfileStore();
+  // const [state, store] = useProfileStore();
+  const [{ ministers }, store] = useLookupStore();
 
   const controller = React.useMemo(
     () => ({
       getMinisters: async () => {
         const response = await dispatch<IMinisterModel[]>('find-all-ministers', () =>
           api.getMinisters(),
         );
-        store.storeMyMinisters(response.data);
+        store.storeMinisters(response.data);
```

Comment:
The ministers are part of the lookups currently.  No need to store them in two places,